test/e2e/framework/service/jig.go

```diff
@@ -957,6 +1136,54 @@ func httpGetNoConnectionPoolTimeout(url string, timeout time.Duration) (*http.Re
 		Transport: tr,
 		Timeout:   timeout,
 	}
-
 	return client.Get(url)
 }
+
+// CreatePausePodDeployment creates a deployment for agnhost-pause pod running in different nodes
+func (j *TestJig) CreatePausePodDeployment(name, ns string, replica int32) *appsv1.Deployment {
+	immediate := int64(0)
```

Comment:
Call this `terminationGracePeriod` but add comment why it's 0 instead?  It's not immediately to me this is used for termination grace period without doing a search for it.

Reply:
Done